README.md

```diff
@@ -1,48 +1,77 @@
-1. A user in fabric8-auth for the serviceaccount used by this app
-1. Users in fabric8-oso-proxy that can be impersonated to create resources on other clusters
-  - These users require `create,delete,watch,get` permissions for `daemonset.apps` in their respective clusters and the specified namespace
-
-To cache images, this app goes through oso-proxy to create daemonsets on desired clusters, which in turn create a pod on each node in the cluster consisting of a list of containers with command `sleep infinity`. This ensures that all nodes in the cluster have those images cached. We also periodically check the health of the daemonset and re-create it if necessary.
+The application can be deployed via Helm or by processing and applying OpenShift Templates.
 
 ## Configuration
-Configuration is done via env vars pulled from `./openshift/configmap.yaml`.
-The config values to be set are
-
-| Env Var | Usage |
-| -- | -- |
-| `CACHING_INTERVAL_HOURS` | Interval, in hours, between checking health of daemonsets |
-| `DAEMONSET_NAME`         | Name of daemonset to be created |
-| `NAMESPACE`              | Namespace where daemonset is to be created. Shared for all users |
-| `IMPERSONATE_USERS`      | Comma-separated list of users to impersonate when creating daemonsets |
-| `OPENSHIFT_PROXY_URL`    | URL of oso-proxy |
-| `IMAGES`                 | List of images to be cached, in the format `<name>=<image>;...` |
-| `OIDC_PROVIDER`          | URL of token provider for service account |
-| `MULTICLUSTER`           | Run in multi cluster mode; default is true |
-
-Additionally, `./openshift/app.yaml` has a few parameters:
-
-| Parameter | Usage |
-| -- | -- |
-| `SERVICEACCOUNT_NAME`             | Name of service account used by main pod |
-| `SERVICE_ACCT_CREDENTIALS_SECRET` | Name of secret storing service account details (see below) |
-| `IMAGE`                           | Name of image used for main pod |
-| `IMAGE_TAG`                       | Tag of image used for main pod |
-
-Finally, a secret containing the pod's serviceaccount's secret and id should be created with the data
-
-| Key | Value |
-| --- | ----- |
-| `service.account.secret` | Service account token |
-| `service.account.id` | User id for service account |
+Configuration is done via env vars pulled from `./deploy/helm/templates/configmap.yaml`, or `./deploy/openshift/configmap.yaml`, depending on the deployment method.
+The config values to be set are:
+
+| Env Var | Usage | Default |
+| -- | -- | -- |
+| `CACHING_INTERVAL_HOURS` | Interval, in hours, between checking health of daemonsets | `"1"` |
+| `CACHING_MEMORY_REQUEST` | The container memory request | `10Mi` |
+| `CACHING_MEMORY_LIMIT` | The container memory limit | `20Mi` |
+| `DAEMONSET_NAME`         | Name of daemonset to be created | `kubernetes-image-puller` |
+| `NAMESPACE`              | Namespace where daemonset is to be created. Shared for all users | `k8s-image-puller` |
```

Comment:
"Shared for all users" only really makes sense downstream (where users ~ clusters)